src/ucs/sys/topo/base/topo.c

```diff
@@ -445,6 +445,7 @@ ucs_topo_get_distance_sysfs(ucs_sys_device_t device1,
     }
 
     /* Report best perf for common PCI bridge or sysfs parsing error */
+    status = ucs_topo_get_distance_default(device1, device2, distance);
```

Comment:
now can we remove line 408 and goto here (restore distance_default label)?

Reply:
Restored it, please check now